stoqs/stoqs/templates/stoqs/stoqsquery.html

```diff
@@ -3239,6 +3236,9 @@
             $('#mp-x3d-geoviewpoint1').attr('position', $('body').data()['extent'][2] + ' ' + $('body').data()['extent'][1] + ' 1000');
             $('#mp-x3d-geoviewpoint1').attr('orientation', '1 0 0 -1.57');
             $(this).html('Enter Cardboard');
+            $('#mp-x3d-geoviewpoint1').attr("set_bind", "false");
+            $(".accordion-inner").css("padding","9px 15px");
```

Comment:
I think it's better to specify the specific elements (IDs) for resetting the css attributes rather than using the class names.